.github/workflows/sca_scan.yaml

```diff
@@ -0,0 +1,31 @@
+name: Snyk Software Composition Analysis Scan
```

Comment:
Suggest renaming the file to something like `snyk_scan.yaml`.

Reply:
Are you suggesting prepending `snyk_` to the name?  This is the naming convention we're using in the other repos as well since there are two different Snyk scans.

Reply:
Yes, I think prefixing the filename with `snyk_` would help.